examples/fonts-builtin/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Using built-in browser fonts
+
+This example shows what happens when auto-loading of fonts from Google
+is turned off and not other fonts are configured:
+
+regular - __bold__ - *italic* - __*bold italic*__ - `code`
+
+You can inspect which fonts are actually used by your browser by using
+the built-in web developer tools (available at least in Chrome, Firefox,
+Safari, Edge) and choosing the fonts view. Try this with multiple
+browsers and compare the results to see if using only built-in fonts
+is an option for you.
+
+## How it works
+
+Turn off the default use of Roboto fonts loaded from Google Fonts in
+your `mkdocs.ym`:
+
+```
+theme:
+  font: false
+```
+
+The fonts used are those that are built into the web browser.
+Material for MkDocs provides sensible defaults that should work in all
+common browsers. 
+
+## (Dis-)Advantages
```

Comment:
I'd say we should mention the privacy plugin to have a "best of both worlds" solution ☺️ Not sure where, but a small admonition that explains the plugin (just take one from our docs) might be good

Reply:
Add the tip - doing the same in the custom fonts example I just pushed a draft of.